src/error.rs

```diff
@@ -1,5 +1,15 @@
-#[cfg(feature = "sqlx-dep")]
-use sqlx::error::Error as SqlxError;
+#[cfg(all(feature = "sea-orm-internal", feature = "sqlx-dep"))]
+pub use sqlx::error::Error as SqlxError;
```

Comment:
This is wrong, as the logic is supposed to be "if we have sqlx-dep, use this symbol; if we have sea-orm-internal, ADD pub"